cfme/tests/containers/test_views_objects.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui import toolbar as tb
+from utils import testgen
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.5"),
+    pytest.mark.usefixtures('setup_provider')]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+
+def test_pods_views():
+    sel.force_navigate('containers_pods')
+    tb.select('Grid View')
+    assert tb.is_active('Grid View'), "Pods grid view setting failed"
```

Comment:
```python
@pytest.mark.parametrize('location', ['containers_pods', 'containers_replicators', ...])
def test_view_setting_whatever_you_like(location):
    sel.force_navigate(location)
    tb.select('Grid View')
    assert tb.is_active('Grid View'), "Replicators grid view setting failed"
    tb.select('Tile View')
    assert tb.is_active('Tile View'), "Replicators tile view setting failed"
    tb.select('List View')
    assert tb.is_active('List View'), "Replicators list view setting failed"
```

This will result in tests:
```
test_view_setting_whatever_you_like[containers_pods]
test_view_setting_whatever_you_like[containers_replicators]
...
```